src/main/java/org/motechproject/newebodac/helper/EncryptionHelper.java

```diff
@@ -0,0 +1,75 @@
+package org.motechproject.newebodac.helper;
+
+import java.security.spec.KeySpec;
+import java.util.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.PBEKeySpec;
+import javax.crypto.spec.SecretKeySpec;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "encryption")
+public class EncryptionHelper {
+
+  /**
+   * Encrypts provided value.
+   */
+  public static String encrypt(String strToEncrypt) {
+
+    /**
+     * Gets required values to encryption - secret key, salt and iv.
+     */
+    String secret = "${encryption.key-crypto}";
```

Comment:
those should be class fields with `@Value` annotation, like you have right now in the KeyController